ranker/modules/omnicorp_overlay.py

```diff
@@ -152,21 +166,32 @@ async def query(request: PDResponse):
         logger.exception(e)
         cache = None
 
-    redis_batch_size = 100
+    dt_start = datetime.now()
+    redis_batch_size = 1000
 
     try:
         # get all node supports.
-
+        start_node_time = datetime.now()
         # first, get the publication counts for all nodes in the kgraph
         keys = list(kgraph["nodes"].keys())
 
         node_pub_counts = {}
+        node_indices = {}
+        index_to_node = {}
+        cache.curie_query(keys)
```

Comment:
This seems to be extra?

Reply:
I think you're correct.  Removed index_to_node.

Reply:
Honestly, I didn't notice that `index_to_node` wasn't being used. My comment was on the `cache.curie_query(keys)` line